firefox-ios/firefox-ios-tests/Tests/XCUITests/ReadingListTests.swift

```diff
@@ -319,6 +319,7 @@ class ReadingListTests: FeatureFlaggedTestBase {
         }
         XCTAssertTrue(app.buttons["Reader View"].isEnabled)
         app.buttons[AccessibilityIdentifiers.Toolbar.addNewTabButton].waitAndTap()
+        mozWaitForElementToExist(app.keyboards.firstMatch)
```

Comment:
We should wait for the keyboard to appear *before* closing the URL bar. When the keyboard has not appear, tapping the `cancelButton` has no effect and then the keyboard appears shortly afterwards.